Src/Eiffel/interface/new_graphical/text_window/text/autocomplete/eb_click_and_complete_tool.e

```diff
@@ -340,6 +340,9 @@ feature -- Basic Operations
 							check is_eiffel_line: False end
 						elseif a_position >= invariant_index then
```

Comment:
If you merge describe_local and describe_feature, it might simplify a few things.